Don't add -D_FORTIFY_SOURCE to the compiler flags, as dpkg-buildflags(1)
already did that for us.

--- a/build/moz.configure/toolchain.configure
+++ b/build/moz.configure/toolchain.configure
@@ -2701,7 +2701,7 @@ def security_hardening_cflags(
         # FORTIFY_SOURCE ------------------------------------
         # Require optimization for FORTIFY_SOURCE. See Bug 1417452
         # Also, undefine it before defining it just in case a distro adds it, see Bug 1418398
-        if compiler_is_gccish and optimize and not asan:
+        if False: # compiler_is_gccish and optimize and not asan:
             flags.append("-U_FORTIFY_SOURCE")
             flags.append("-D_FORTIFY_SOURCE=2")
 
